app/realty/models.py

```diff
@@ -9,12 +9,36 @@ class Flat(models.Model):
         ('Sold', 'Продана')
     ]
 
-    square = models.FloatField(verbose_name='Общая площадь', validators=[MinValueValidator(limit_value=0)])
-    living_space = models.FloatField(verbose_name='Жилая площадь', validators=[MinValueValidator(limit_value=0)])
-    kitchen_area = models.FloatField(verbose_name='Площадь кухни', validators=[MinValueValidator(limit_value=0)])
-    rooms = models.PositiveSmallIntegerField(verbose_name='Количество комнат')
-    floor = models.PositiveSmallIntegerField(verbose_name='Этаж')
-    status = models.CharField(max_length=7, choices=STATUS_CHOICES, default='On sale', verbose_name='Статус')
-    price = models.PositiveBigIntegerField(verbose_name='Цена')
-    description = models.TextField(verbose_name='Описание')
-    photo = models.ImageField(upload_to='photos/%Y/%m/%d/')
+    square = models.FloatField(
+        verbose_name='Общая площадь',
+        validators=[MinValueValidator(limit_value=0)]
+        )
```

Comment:
Эту скобку лучше еще пододвинуть)

```suggestion
    )
```